source/config/Config.cpp

```diff
@@ -1133,12 +1176,19 @@ bool PlainConfig::PubSub::Validate() const
     }
     if (subscribeFile.has_value() && !subscribeFile->empty())
     {
-        if (!FileUtils::ValidateFilePermissions(subscribeFile.value(), Permissions::PUB_SUB_FILES, true))
+        if (FileUtils::IsValidFilePath(subscribeFile->c_str()))
+        {
+            if (!FileUtils::ValidateFilePermissions(subscribeFile.value(), Permissions::PUB_SUB_FILES, true))
+            {
+                LOGM_ERROR(
```

Comment:
This log statement is redundant. We already log errors in `ValidateFilePermissions` method.